pgAgent/build.sh

```diff
@@ -30,15 +30,15 @@ _prep_pgAgent() {
     cd $WD/pgAgent/source
 
     # pgAgent
-    if [ -e pgAgent-$PG_VERSION_PGAGENT-Source ];
+    if [ -e pgagent-pgagent-$PG_VERSION_PGAGENT ];
     then
-      echo "Removing existing pgAgent-$PG_VERSION_PGAGENT-Source source directory"
-      rm -rf pgAgent-$PG_VERSION_PGAGENT-Source  || _die "Couldn't remove the existing pgAgent-$PG_VERSION_PGAGENT-Source source directory (source/pgAgent-$PG_VERSION_PGAGENT-Source)"
+      echo "Removing existing pgagent-pgagent-$PG_VERSION_PGAGENT source directory"
+      rm -rf pgagent-pgagent-$PG_VERSION_PGAGENT  || _die "Couldn't remove the existing pgagent-pgagent-$PG_VERSION_PGAGENT source directory (source/pgagent-pgagent-$PG_VERSION_PGAGENT)"
     fi
 
     echo "Unpacking pgAgent source..."
-    extract_file  ../../tarballs/pgAgent-$PG_VERSION_PGAGENT-Source 
-    cd pgAgent-$PG_VERSION_PGAGENT-Source
+    extract_file  ../../tarballs/pgagent-$PG_VERSION_PGAGENT 
```

Comment:
from what location did you download that source? because I still see the source with the old name at https://www.pgadmin.org/download/pgagent-source-code/